shell/platform/linux/fl_pixel_buffer_texture.cc

```diff
@@ -0,0 +1,88 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/linux/public/flutter_linux/fl_pixel_buffer_texture.h"
+
+#include <epoxy/gl.h>
+#include <gmodule.h>
+#include <cstdio>
+
+#define GL_CALL(expr)                                                      \
+  do {                                                                     \
+    expr;                                                                  \
+    GLenum err = glGetError();                                             \
+    if (err) {                                                             \
+      fprintf(stderr, "glGetError %x (%s:%d)\n", err, __FILE__, __LINE__); \
+    }                                                                      \
+  } while (0);
```

Comment:
Macros are strongly discouraged by the style guide. AFAICT this could just be a regular function that doesn't include `expr` and is dropped after each call, with __LINE__ as an argument.